master/reference/calicoctl/resources/ippool.md

```diff
@@ -56,6 +58,12 @@ Hosts running {{site.prodname}} is asymmetric and may cause traffic to be filter
 [RPF](https://en.wikipedia.org/wiki/Reverse_path_forwarding) checks failing.
 {: .alert .alert-info}
 
+#### Block Sizes
+The default block sizes of `26` for IPv4 and `122` for IPv6 provide blocks of 64 addresses. This allows addresses to be allocated in groups to workloads running on the same host. By grouping addresses, fewer routes need to be exchanged between hosts and to other BGP peers. If a host allocates all of the addresses in a block then it will be allocated an additional block. If there are no more blocks available then the host can take addresses from blocks allocated to other hosts. Specific routes are added for the borrowed addresses which has an impact on route table size.
+
+Increasing the block size from the default (e.g. using `24` for IPv4 to give 256 addresses per block) means fewer blocks per host, and potentially fewer routes. But try to ensure that there are at least as many blocks in the pool as there are hosts.
```

Comment:
`e.g.` and `i.e.` should always be enclosed in commas. In this case, with the opening parenthesis, just a comma after `e.g.` will do it. Line 64 and 66.